app/src/main/res/navigation/navigation_temp.xml

```diff
@@ -2,8 +2,8 @@
 <navigation xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/navigation_temmp"
-    app:startDestination="@id/splashFragment">
+    android:id="@+id/navigation_temp"
```

Comment:
Should we rename this?

Reply:
will do so in "cleanup" PR, where I will delete unused resources and files